Rover/AP_Arming.cpp

```diff
@@ -150,7 +152,9 @@ bool AP_Arming_Rover::disarm(const AP_Arming::Method method, bool do_disarm_chec
 
     update_soft_armed();
 
-    gcs().send_text(MAV_SEVERITY_INFO, "Throttle disarmed");
+    if (!option_enabled (AP_Arming::Option::DISABLE_STATUSTEXT_ON_STATE_CHANGE)) {
+        gcs().send_text(MAV_SEVERITY_INFO, "Throttle disarmed");
+    }
```

Comment:
```suggestion
   send_arm_disarm_statustext(MAV_SEVERITY_INFO, "Throttle disarmed");
```

... and that method looks at the arming bit.  Much smaller patch, probably save some flash

Reply:
This was done